Script/AtomicNET/AtomicNET/Atomic2D/PhysicsWorld2D.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace AtomicEngine
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct PhysicsRaycastResult2D
+    {
+        public Vector2 Position;
+        public Vector2 Normal;
+        public float Distance;
+
+        IntPtr bodyPtr;
+        public RigidBody2D Body => bodyPtr == IntPtr.Zero ? null : NativeCore.WrapNative<RigidBody2D>(bodyPtr);
+    }
```

Comment:
C# is so fancy, this raw access to NativeCore is something to look at some day soon.  It would be good to have a cleanup pass at Atomic NET, the code literally flew together in some places ;)